pkg/runtime.v2/framework/plugins/coscheduling/coscheduling.go

```diff
@@ -87,19 +88,18 @@ func (c *CoScheduling) Name() string {
 	return Name
 }
 
-func (c *CoScheduling) EnforcePodGroupPolicy(trainJob *kubeflowv2.TrainJob, info *runtime.Info) error {
-	if info == nil || info.PodGroupPolicy == nil || trainJob == nil {
+func (c *CoScheduling) EnforcePodGroupPolicy(info *runtime.Info, trainJob *kubeflowv2.TrainJob) error {
+	if info == nil || info.RuntimePolicy.PodGroupPolicy == nil || trainJob == nil {
 		return nil
 	}
-	if info.PodLabels == nil {
-		info.PodLabels = make(map[string]string, 1)
-	}
-	info.PodLabels[schedulerpluginsv1alpha1.PodGroupLabel] = trainJob.Name
+
+	info.Scheduler.PodLabels = make(map[string]string, 1)
```

Comment:
What is the reason that we forcefully override the Pod labels?

Reply:
We only use `PodLabels` in the Scheduler right now, that is why we just creating a new Map here.
In the future, if we should do any Pod labels merge for the Info object, we should revisit it.

Reply:
Basically, this is pluggabble mechanism, so we should consider the other plugin.
So, as before initialization impl, we should initialize map only when the pod label is emtpy